src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    subtract(value) {
+      this.result -= value;
+
+      return this.result;
+    },
+
+    add(value) {
+      this.result += value;
+
+      return this.result;
```

Comment:
Similarly, the return statement in the `add` method is unnecessary and can be removed.